utils/linkage.py

```diff
@@ -1,9 +1,12 @@
 import textdistance as td
-import usaddress
+
+# import usaddress
```

Comment:
this needs to be uncommented presumably for the other functions

Reply:
Uncommented and put the address function back in to prevent linter errors.